designer/client/src/hooks/list/useListItem/useListItem.tsx

```diff
@@ -89,6 +89,14 @@ export function useListItem(
       schema
     })
 
+    if (selectedList?.type === 'number') {
+      errors.value ??= validateCustom('value', value, {
+        message: 'errors.number',
+        label: `Item value '${value}'`,
+        schema: Joi.number().required()
```

Comment:
This is looking ace @davidjamesstone

I'm using dependency injection for Joi now so if you can remove `import Joi` and change this line?

Then webpack can lazy-load `/javascripts/vendor/joi.[hash].min.js` via https://github.com/DEFRA/forms-designer/commit/1c3370aec473c00402c680c116be7c6fc599f55a

Hadn't realised the Joi browser build was 144KB (see https://github.com/DEFRA/forms-designer/pull/497 for info)

```suggestion
        schema: schema.number().required()
```

Reply:
Not sure how I missed that. During the rebase maybe.

Fixed in 2331da9c3618413dada7df4aaa1c215488cb7f21

Reply:
Yeah joi.browser is a bit big, I believe they already strip a bunch of email/URL logic out to bring the file size down a bit.